src/auth/core/redux/actions.ts

```diff
@@ -272,3 +271,36 @@ export const RegisterUser = (registerData: RegisterData) => async (
     console.log("error in try", e);
   }
 };
+
+export const EmailVerification = (token: string) => async (
+  dispatch: Dispatch<LoginDispatchTypes>
+) => {
+  try {
```

Comment:
action should not have async code. move this to redux-saga or redux-thunk

Reply:
Okay. Made changes accordingly in all action functions.